#!/usr/bin/python3
"""6-raise_exception_msg
Raise a NameError with a custom message.
"""


def raise_exception_msg(message=""):
    """Raise NameError containing *message*."""
    raise NameError(message)
